apps/frontend/src/app/3_organisms/ConversionsHistoryFrame/hooks/useGetConversionsHistory.ts

```diff
@@ -0,0 +1,40 @@
+import { useQuery } from '@apollo/client';
+
+import { useMemo } from 'react';
+
+import { OrderOptions } from '@sovryn/ui';
+
+import { myntClient } from '../../../../utils/clients';
+import {
+  Conversion_OrderBy,
+  GetUserConversionsDocument,
+} from '../../../../utils/graphql/mynt/generated';
+
+export const useGetConversionsHistory = (
+  account: string,
+  pageSize: number,
+  page: number,
+  orderOptions: OrderOptions,
+) => {
+  const config = useMemo(
+    () => ({
+      user: account,
+      skip: page * pageSize,
+      pageSize,
+      orderBy: orderOptions.orderBy as Conversion_OrderBy,
+      orderDirection: orderOptions.orderDirection,
+    }),
+    [
+      account,
+      orderOptions.orderBy,
+      orderOptions.orderDirection,
+      page,
+      pageSize,
+    ],
+  );
+
+  return useQuery(GetUserConversionsDocument, {
+    variables: config,
+    client: myntClient,
+  });
```

Comment:
I couldn't because I needed to pass a custom client (`myntClient`) but since I'd need it for CSV export as well, I decided to wrap the component in `ApolloProvider` and pass the custom client there.